### DISCUSSION TITLE
I found a post which is really nice to understand Telegraf

### DISCUSSION THREAD

**Discussion by MAKAR190** [telegraf/telegraf#1586] [General]
Here it is:
[Dev.To post](https://dev.to/maklut/telegraf-vs-node-telegram-bot-api-36fk)
It solves problems which I had, like WizardScene chain, registering scenes, session etc. Mb it will help you...

**Comment by MKRhere** [comment]
Thanks for sharing!

**Reply by MAKAR190** [reply]
> Thanks for sharing!

Your welcome 🙂
